src/DotLiquid/MIISdotliquid.csproj

```diff
@@ -6,7 +6,7 @@
     <NeutralLanguage>en-US</NeutralLanguage>
     <Authors>Tim Jones;Alessandro Petrelli</Authors>
     <TargetFrameworks>netstandard1.3;net45</TargetFrameworks>
-    <AssemblyName>DotLiquid</AssemblyName>
+    <AssemblyName>MIISdotliquid</AssemblyName>
```

Comment:
Assembly rename merge to master... Why?

Reply:
This is a later commit I did and wasn't in the original pull request 😞 

Since nobody seemed to answer the original pull request and I needed this fixed to use it in my project, I decided to use my fork as a new project, independent of your original DotNetLiquid. To use a different name and being able to distribute it with my other project (MIIS), I changed the Assembly name. I'd prefer to use just DotLiquid and not my own version, but since nobody seemed to be in the project anymore...

This can be ignored if you decide to incorporate the fix (and the first commit) and I happily will use DotLiquid instead.